cmake/projects/PostgreSQL/hunter.cmake

```diff
@@ -18,6 +19,7 @@ hunter_add_version(
     b82ddcee4644ef42f3a69ee93916afa448d178c4
 )
 
+hunter_configuration_types(PostgreSQL CONFIGURATION_TYPES Release)
 hunter_pick_scheme(DEFAULT url_sha1_autotools)
 hunter_download(PACKAGE_NAME PostgreSQL)
```

Comment:
This package is not cacheable, do I still need to have the `PACKAGE_INTERNAL_DEPS_ID` for it? Maybe something changed, but I thought I had to add to cacheable packages only